### PR TITLE
BookStore: Set up

### PR DESCRIPTION
**Pull request by jahongiry**
**Added basic skeleton**
- Added React Router
- Added two routes (Books as default and Categories)
- No styling (since it was not required)

### CODE REVIEW

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @jahongiry,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please edit your README file by using [the Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and edit it with your personal and project info.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
```
Comment (chrystalme) [inline package.json:19]: Kindly note that this is the script for running tests on your `react app` and it is run by `npm run test`

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+
+```
+ npm start
+```
+
+### Run tests
+
+- For running project
+
+`npm start`
```
Comment (chrystalme) [inline README.md]: Kindly note that to run tests for `react app` you run `npm run test`

File: README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+  ```
+    npm run lint:es
+  ```
+
+## Authors
+
+👤 Jahongir Yusupov
+
+- GitHub: [https://github.com/jahongiry](https://github.com/jahongiry)
+- LinkedIn: [linkedin.com/in/jahongir-yusupov](linkedin.com/in/jahongir-yusupov)
```
Comment (chrystalme) [inline README.md:57]: Kindly properly customise the handles to reflect your name and not the URL to your page. example `-GitHub : [@jahongiry](https://github.com/jahongiry)`

File: README.md [hunk 4]
```diff
@@ -1 +1,74 @@
...
+### Prerequisites
+
+- Must have a working computer
+- A Web-Browser installed (Chrome - Recommended)
+
+### Setup
+
+- Install _VSCode_ ([link](https://code.visualstudio.com/download))
+- Clone this project repository on your local computer. ([link](https://github.com/jahongiry/Bookstore-REACT_REDUX.git))
+- Open the project directory with _VSCode_
```
Comment (chrystalme) [inline README.md]: Your setup does not guide the individual on how to get your project running locally. 
## Steps
 - `$ git clone ...`
 - `$ cd directory_of_project`
 - `$ git checkout branch_to_check or git switch branch_to_check`
 - `$ npm install`
 - `$ npm start`

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @jahongiry, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 💡 
✔️ Linters properly setup
✔️ Project looks tidy having removed unwanted files and folders
## Suggested changes ♻️ 
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @jahongiry, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**